internal/containerupdate/fake_container_updater.go

```diff
@@ -22,6 +26,15 @@ type UpdateContainerCall struct {
 	HotReload  bool
 }
 
+func (cu *FakeContainerUpdater) ValidateSpecs(specs []model.TargetSpec, env k8s.Env) error {
+	var err error
```

Comment:
if you wanna golf, this is all the same as:
```
err := cu.ValidateErr
cu.ValidateErr = nil
return err
```

Reply:
done